src/extension/providers/nostr/types.ts

```diff
@@ -15,7 +15,25 @@ export enum EventKind {
   Contacts = 3,
   DM = 4,
   Deleted = 5,
+  Repost = 6,
```

Comment:
what are these added types used for and where?

Reply:
permission will be set for this kinds in one click while selecting reasonable option in enable screen

Reply:
but in enable.ts above, they are not used (you use numbers again and with comments to describe the number)